### PR TITLE
Validation of initial ConfigSettingEntries

### PR DESCRIPTION
**Pull request by sisuresh**
# Description

The LedgerEntryIsValid invariant is only checked after operation application, so the ConfigSettingEntry case is never hit. This change validates the initial settings with the same checks that we do during upgrades. 

# Checklist
- [ ] Reviewed the [contributing](https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#submitting-changes) document
- [ ] Rebased on top of master (no merge commits)
- [ ] Ran `clang-format` v8.0.0 (via `make format` or the Visual Studio extension)
- [ ] Compiles
- [ ] Ran all tests
- [ ] If change impacts performance, include supporting evidence per the [performance document](https://github.com/stellar/stellar-core/blob/master/performance-eval/performance-eval.md)

### CODE REVIEW

**Comment by dmkozh** [comment]
r+ f0ded7a

**Comment by dmkozh** [comment]
r+ 2f92171

File: src/overlay/test/OverlayTests.cpp [hunk 1]
```diff
@@ -35,6 +35,43 @@ using namespace stellar::overlaytestutils;
...
+    ClaimPredicate andPred;
+    andPred.type(CLAIM_PREDICATE_AND);
+    andPred.andPredicates().emplace_back(recursivePredicate(counter + 1));
+    andPred.andPredicates().emplace_back(recursivePredicate(counter + 1));
+
+    return andPred;
+}
+
+Operation
+getOperationGreateThanMinMaxSizeBytes()
```
Comment (marta-lokhova) [inline src/overlay/test/OverlayTests.cpp:57]: nit: getOperationGreaterThanMinMaxSizeBytes

**Review by marta-lokhova — COMMENTED** [review]
LGTM!

**Pull request merged**